src/main/java/movie/MoviesByFactors.java

```diff
@@ -0,0 +1,91 @@
+package movie;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import runner.AppRunner;
+
+public class MoviesByFactors {
+
+  private static final int CATEGORY = 1;
+  private static final int STATUS = 2;
+  private static final int TYPE = 3;
+  private static final int BACK = 4;
+  private static final int EXIT = 5;
+  private static Scanner scan = new Scanner(System.in);
+  private static int input;
+
+  public static void sortByFactor() {
+
+    input = scan.nextInt();
+
+    while (input != EXIT) {
+      switch (input) {
+        case CATEGORY:
+          List<String> drama = MovieCreator.movies.stream()
+              .filter(movie -> movie.getCategory()
```

Comment:
You have to refactor the whole this class. Don't repeat yourself.

Reply:
fixed